hat/assets/js/apps/Iaso/domains/stock/versions/components/VersionBaseInfo.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import {
+    Table,
+    TableBody,
+    TableRow,
+    TableCell,
+    Divider,
+    Box,
+} from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import { useSafeIntl } from 'bluesquare-components';
+
+import { DetailsForm } from 'Iaso/domains/stock/versions/components/DetailsForm';
+import MESSAGES from '../../messages';
+
+import { StockRulesVersion } from '../../types/stocks';
+
+import { PublishVersionModal } from './PublishVersionModal';
+import { StatusCell } from './StatusCell';
+
+const useStyles = makeStyles(theme => ({
+    leftCell: {
+        // @ts-ignore
+        borderRight: `1px solid ${theme.palette.ligthGray.border}`,
+        fontWeight: 'bold',
+    },
+}));
+
+type RowProps = {
+    label: string;
+    value?: string | ReactNode;
+};
+
+const Row: FunctionComponent<RowProps> = ({ label, value }) => {
+    const classes = useStyles();
+    return (
+        <TableRow>
+            <TableCell className={classes.leftCell}>{label}</TableCell>
+            <TableCell>{value}</TableCell>
+        </TableRow>
+    );
+};
+
+type Props = {
+    version: StockRulesVersion;
+};
+export const VersionBaseInfo: FunctionComponent<Props> = ({ version }) => {
+    const { formatMessage } = useSafeIntl();
+    return (
+        <>
+            <DetailsForm version={version} />
+            <Divider />
+            <Table size="small" data-test="workflow-base-info">
+                <TableBody>
+                    <Row
+                        label={formatMessage(MESSAGES.status)}
+                        value={
+                            version ? (
+                                <StatusCell status={version.status} />
+                            ) : (
+                                ''
+                            )
```

Comment:
You can handle this directly in the Row component . Also i would use placeholder from bluesquare-component

Reply:
I don't understand the "You can handle this directly in the Row component" part